### PR TITLE
feat: update how to integrate

### PR DESCRIPTION
**Pull request by takayuki5168**
Signed-off-by: Takayuki Murooka <takayuki5168@gmail.com>

## Description

Add a explanation of how to integrate Autoware to your vehicle, considering config in the `autoware_launch` package described here
https://github.com/autowarefoundation/autoware_launch/issues/66

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md [hunk 1]
```diff
@@ -166,8 +192,12 @@ The panel can be found in `Panels > Add New Panel > tier4_state_rviz_plugin > Au
...
 Now the vehicle should drive along the calculated path!
 
-## 7. Tune parameters for your vehicle & environment
+## 8. Tune parameters for your vehicle & environment
 
 You may need to tune your parameters depending on the domain in which you will operate your vehicle.
 
+Note that there are some kinds of parameter files in Autoware.
+The parameter files you should modify for integration is in your forked Autoware Launch repository cloned in Section. 4.
+You can see more deail about the parameter management in Autoware [here](https://autowarefoundation.github.io/autoware-documentation/pr-171/contributing/coding-guidelines/ros-nodes/parameters/).
```
Comment (LalithVipulananthan) [inline docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md:201]: Need to fix typo. Also, the link to the coding guidelines page for parameters should be relative I think.

File: docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md [hunk 2]
```diff
@@ -41,7 +41,33 @@ git clone https://github.com/YOUR_NAME/autoware.git
 
 If you set up multiple types of vehicles, adding a suffix like "autoware.vehicle_A" or "autoware.vehicle_B" is recommended.
 
-## 4. Create the description packages of your vehicle
+## 4. Create your Autoware Launch repository
```
Comment (LalithVipulananthan) [inline docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md:44]: The repository is referred to as `Autoware Launch` a few times (see lines 57 and 200), but the README file says `autoware_launch`. Which name is correct?

**Review by LalithVipulananthan — COMMENTED** [review]
A quick review since there are only a few changes.

File: docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md (hunk 1 above)
Comment (kenji-miyake) [inline docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md:201]: typo

```suggestion
You can see more detail about the parameter management in Autoware [here](https://autowarefoundation.github.io/autoware-documentation/pr-171/contributing/coding-guidelines/ros-nodes/parameters/).
```

**Comment by xmfcx** [comment]
@takayuki5168 could you go over the reviews and also rebase it to the main branch?

**Comment by takayuki5168** [comment]
I don't have much time for this. So I close the PR.

**Pull request closed without merging**